src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/DateAttributeConfigurationType.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Form\Type\AttributeType;
```

Comment:
This form types could go to separate namespace here ``Configuration``, otherwise we will have quite a lot of classes here. :)